Library/Homebrew/dev-cmd/cat.rb

```diff
@@ -20,7 +20,7 @@ def cat
     cat_args.parse
 
     cd HOMEBREW_REPOSITORY
-    pager = if Homebrew::EnvConfig.bat?
+    pager = if ENV["BAT_CONFIG_PATH"] = Homebrew::EnvConfig.bat_config_path
```

Comment:
Use `Homebrew::EnvConfig.bat?` and set `ENV["BAT_CONFIG_PATH"]` if `Homebrew::EnvConfig.bat_config_path` is set (and same for `cask/cmd/cat` above).